charades-ui/src/routes/root-router.tsx

```diff
@@ -0,0 +1,8 @@
+import Auth from '@components/auth';
+import { PATH } from '@utils/paths';
+import { RouteObject } from 'react-router-dom';
+
+export const RootRouter: RouteObject = {
+  path: PATH.ANY,
```

Comment:
Why not `PATH.ROOT`?

Reply:
Because it catches everything `*` right now and returns RootRouter